ios/FluentUI.Demo/FluentUI.Demo/DemoHostingController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import FluentUI
+
+/// A specialized subclass of `FluentThemedHostingController` that can be used for SwiftUI demo screens.
+class DemoHostingController: FluentThemedHostingController {
+    init(rootView: AnyView, title: String) {
+        super.init(rootView: rootView)
+        self.title = title
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    @MainActor required dynamic init(rootView: AnyView) {
+        super.init(rootView: rootView)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureAppearanceAndReadmePopovers()
+    }
+
+    // MARK: - Demo Appearance Popover
+
+    func configureAppearanceAndReadmePopovers() {
```

Comment:
The name is `configureAppearanceAndReadmePopovers` but there is no readme popover anywhere in the code or the file. Are we intentionally omitting the readme vc here?

Reply:
Good catch--I never added the `ReadmeController` to the Button's SwiftUI controller, so that lack was inherited here. Adding it now!